src/meta/MetaServiceUtils.cpp

```diff
@@ -1000,6 +1007,28 @@ std::string MetaServiceUtils::genTimestampStr() {
     return ch;
 }
 
+ErrorOr<nebula::cpp2::ErrorCode, bool> MetaServiceUtils::isIndexRebuilding(
+    kvstore::KVStore* kvstore) {
+    folly::SharedMutex::ReadHolder rHolder(LockUtils::spaceLock());
```

Comment:
Could reuse StorageEnv::getIndexState to check the index state. This one is probably outdated.

Reply:
Does this mean we need to check in storage to see if the index was rebuilding?

Reply:
Sorry I didn't see it is meta here. But it is outdated definitely. If you track the code, actually `rebuildIndexStatusPrefix` is not written at all.

Reply:
Maybe you could add a method in job manager to check if job in `inFlightJobs_`.

Reply:
Okay, I've created a new JobManager::checkIndexJobRunning method inside the new commit to do this.